### DISCUSSION TITLE
Novo logo da FCT -> Nova School of Sciences and Technology

### DISCUSSION THREAD

**Discussion by jpdelopes** [joaomlourenco/novathesis#161] [Ideas]
Hi João, 

Will the new naming and logo of FCT (Nova School of Sciences and Technology) / color scheme be included on the template soon?

Thankfully, 
JL

**Comment by joaomlourenco** [comment]
Please use English in this forum! Thanks! 👍

It is not up to me to decide when the new logos and color schemes will become effective in the covers and insides of the thesis and dissertation layouts. I'm just the author of the LaTeX template... 😊

But I'm working with the FCT-NOVA services to have a version of the LaTeX template ready when the new rules and visual are announced. 

I may finish by saying that _I believe_ it will happen until the end of this month. But this is only a belief. 

Finally, if you are using the LaTeX template v5.x, changing will be as simple as changing the name of the school for another name. 😃

**Reply by jpdelopes** [reply]
Thank you for your reply, 

I've edited by post to English, thank you for the warning.

I will wait then 😉
